### ISSUE TITLE
Plotting related --no-gpu and --no-cpu flags raise AttributeError

### ISSUE THREAD

**Issue by balazsfabian** [bio-phys/MDBenchmark#177]
**Expected behavior**

```bash
mdbenchmark plot --csv dataset1.csv --csv dataset2.csv --output-format pdf --no-gpu
```
Should produce a performance plot without gpu.

**Actual behavior**
```bash
Traceback (most recent call last):
  File "/u/bafabian/conda-envs/py310/bin/mdbenchmark", line 11, in <module>
    sys.exit(cli())
  File
"/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/click/core.py",
line 1130, in __call__
    return self.main(*args, **kwargs)
  File
"/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/click/core.py",
line 1055, in main
    rv = self.invoke(ctx)
  File
"/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/click/core.py",
line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File
"/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/click/core.py",
line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File
"/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/click/core.py",
line 760, in invoke
    return __callback(*args, **kwargs)
  File "/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/mdbenchmark/cli/commands.py",
line 353, in plot
    do_plot(
  File "/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/mdbenchmark/cli/plot.py",
line 236, in do_plot
    df = filter_dataframe_for_plotting(df, template, module, gpu, cpu)
  File "/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/mdbenchmark/cli/plot.py",
line 159, in filter_dataframe_for_plotting
    df = df[~df.gpu]
  File "/u/bafabian/conda-envs/py310/lib/python3.10/site-packages/pandas/core/generic.py",
line 5902, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'gpu'
```

The same error when using the ```--no-cpu``` flag, the attribute is still 'gpu'.

**Your version of MDBenchmark:**
mdbenchmark, version 3.0.1

**Comment by ksebaz** [comment]
This should be fixed with #192.

Would you mind to test that with the develop-branch @balazsfabian?

**Comment by balazsfabian** [comment]
The issue is fixed with #192.

**Comment by ksebaz** [comment]
Thank you for checking!
